### ISSUE TITLE
[Bug]: Pyarrow is required when importing towhee

### ISSUE THREAD

**Issue by binbinlv** [towhee-io/towhee#2108]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Pyarrow is required when importing towhee

### Expected Behavior

Pyarrow is not required when importing towhee

### Steps To Reproduce

```markdown
1. install towhee latest 
2. import towhee
```

### Environment

```markdown
- Towhee version(e.g. v0.1.3 or 8b23a93):towhee latest (0.8.1.dev354)
- OS(Ubuntu or CentOS):
- CPU/Memory:
- GPU:
- Others:
```

**Comment by binbinlv** [comment]
![fHceiPrmIg](https://user-images.githubusercontent.com/83755740/191713515-82a99e8c-d1e3-425f-b3bc-7e57acfb7182.jpg)

**Comment by filip-halt** [comment]
This is being caused by `towhee.types.tensor_array.py`. Tensor array is imported in __init__ of towhee.types, causing the immediate install prompt on import towhee. @Chiiizzzy

**Comment by binbinlv** [comment]
Verified, and it is fixed in the latest towhee versioin (0.8.1.dev389),
